tests/test_utils.py

```diff
@@ -1321,6 +1321,7 @@ def a_method(self, a=1, b=2):
         assert len(recwarn) == 0
         C().a_method(1, b=2)
         assert len(recwarn) == 1
+        assert 'a_method(...)' in str(recwarn[0].message)
```

Comment:
Could we change this assertion to ensure that `C.` from `__qualname__` is surfaced in the message as well?

Reply:
done